Formula/mingw-w64.rb

```diff
@@ -0,0 +1,225 @@
+class MingwW64 < Formula
+  desc "Minimalist GNU for Windows with GCC."
+  homepage "https://mingw-w64.org/"
+  # https://sourceforge.net/p/mingw-w64/wiki2/Cross%20Win32%20and%20Win64%20compiler/
+  url "https://downloads.sourceforge.net/project/mingw-w64/mingw-w64/mingw-w64-release/mingw-w64-v5.0.1.tar.bz2"
+  sha256 "9bb5cd7df78817377841a63555e73596dc0af4acbb71b09bd48de7cf24aeadd2"
+
+  option "without-i686", "Compile without i686 compiler"
+  option "without-x86_64", "Compile without x86_64 compiler"
+  # option "without-multilib", "Compile x86_64 copmiler without multilib support"
+
+  depends_on "texinfo" => :build
+  depends_on "mpfr"
+  depends_on "libmpc"
+  depends_on "isl"
+  depends_on "gmp"
+  depends_on "wine" => :optional
+
+  resource "binutils" do
+    url "https://ftpmirror.gnu.org/binutils/binutils-2.27.tar.gz"
+    sha256 "26253bf0f360ceeba1d9ab6965c57c6a48a01a8343382130d1ed47c468a3094f"
+  end
+
+  resource "gcc" do
+    url "https://ftpmirror.gnu.org/gcc/gcc-6.3.0/gcc-6.3.0.tar.bz2"
+    sha256 "f06ae7f3f790fbf0f018f6d40e844451e6bc3b7bc96e128e63b09825c1f8b29f"
+  end
+
+  def target_archs
+    if build.without?("i686") && build.without?("x86_64")
+      odie "Options --without-i686 and --without-x86_64 are mutually exclusive"
+    end
+    archs = []
+    archs.push("i686-w64-mingw32") if build.with?("i686")
+    archs.push("x86_64-w64-mingw32") if build.with?("x86_64")
+  end
+
+  def install
+    target_archs.each do |target_arch|
+      resource("binutils").stage do
```

Comment:
If we decide to depends on `mingw-w64-bintulis` in this formula, we can remove `resource("binutils")` block from here.